TranscriberModels.py

```diff
@@ -9,22 +9,34 @@ def get_model(use_api):
     else:
         return WhisperTranscriber()
 
-class WhisperTranscriber:
-    def __init__(self):
+class WhisperTranscriber():
+    
+    
+    def __init__(self, language="en"):
+        self.lang = language
         self.audio_model = whisper.load_model(os.path.join(os.getcwd(), 'tiny.en.pt'))
         print(f"[INFO] Whisper using GPU: " + str(torch.cuda.is_available()))
 
     def get_transcription(self, wav_file_path):
         try:
-            result = self.audio_model.transcribe(wav_file_path, fp16=torch.cuda.is_available())
+            result = self.audio_model.transcribe(wav_file_path, fp16=torch.cuda.is_available(), language = self.lang)
         except Exception as e:
             print(e)
             return ''
         return result['text'].strip()
     
+    def change_lang(self,language):
+        self.lang = language
+        if self.lang == "en":
+            self.audio_model = whisper.load_model(os.path.join(os.getcwd(), 'tiny.en.pt'))
+        else:
+            self.audio_model = whisper.load_model(os.path.join(os.getcwd(), 'tiny.pt')) 
+
 class APIWhisperTranscriber:
     def get_transcription(self, wav_file_path):
-        audio_file= open(wav_file_path, "rb")
+        new_file_path = wav_file_path + '.wav'
```

Comment:
please rebase your branch, should not these changes unrelated to this feature in your PR